File: docs/misc/codesnippet/CSharp/how-to-program-the-designer-region-of-the-status-bar_1.cs
        void DisplayDesignerRegionInfo(IVsStatusbar statusBar,
            bool insert, int line, int column, int character)
        {
            // Set insert/overstrike mode.
            object mode = insert ? 0 : 1;
            statusBar.SetInsMode(ref mode);

            // Display Ln ## Col ## Ch ## information.
            object ln = line, col = column, ch = character;
            statusBar.SetLineColChar(ref ln, ref col, ref ch);
        }

File: docs-old-1104/misc/codesnippet/VisualBasic/how-to-program-the-designer-region-of-the-status-bar_1.vb
    Private Sub DisplayDesignerRegionInfo(ByVal statusBar As IVsStatusbar, ByVal insert As Boolean, ByVal line As Integer, ByVal column As Integer, ByVal character As Integer)
        ' Set insert/overstrike mode. 
        Dim mode As Object = If(insert, 0, 1)
        statusBar.SetInsMode(mode)

        ' Display Ln ## Col ## Ch ## information. 
        Dim ln As Object = line, col As Object = column, ch As Object = character
        statusBar.SetLineColChar(ln, col, ch)
    End Sub